src/main/java/org/jenkinsci/plugins/deployment/workflowsteps/AwaitDeploymentStepExecution.java

```diff
@@ -0,0 +1,66 @@
+package org.jenkinsci.plugins.deployment.workflowsteps;
+
+import com.google.inject.Inject;
+import hudson.model.Fingerprint;
+import hudson.model.Job;
+import hudson.model.Run;
+import org.jenkinsci.plugins.deployment.DeploymentFacet;
+import org.jenkinsci.plugins.deployment.HostRecord;
+import org.jenkinsci.plugins.deployment.conditions.ThresholdCondition;
+import org.jenkinsci.plugins.workflow.graph.FlowNode;
+import org.jenkinsci.plugins.workflow.steps.AbstractStepExecutionImpl;
+import org.jenkinsci.plugins.workflow.steps.StepContextParameter;
+
+import java.util.concurrent.ScheduledFuture;
+
+/**
+ * The Execution for AwaitDeploymentStep
+ */
+public class AwaitDeploymentStepExecution extends AbstractStepExecutionImpl {
+
+    @Inject(optional=true) private transient AwaitDeploymentStep awaitDeploymentStep;
+    @StepContextParameter
+    private transient FlowNode node;
+    @StepContextParameter
+    transient Run run;
+    private transient volatile ScheduledFuture<?> task;
+    @StepContextParameter
+    private transient Run build;
+    @StepContextParameter
+    private transient Job job;
```

Comment:
All the above block is really hard to read. Both inline and top-line annotations, no new-lines between definitions, ...